client/my-sites/checkout/checkout-thank-you/jetpack-checkout-thank-you.tsx

```diff
@@ -75,8 +75,8 @@ const JetpackCheckoutThankYou: FunctionComponent< Props > = ( {
 					>
 						{ translate( '%(productName)s was added to your site %(siteName)s.', {
 							args: {
-								productName,
-								siteName,
+								productName: productName as string, // We know this exists via hasProductInfo
+								siteName: siteName ?? '',
```

Comment:
If `siteName` is nullish, the string will end up pretty awkwardly with a space and a dot. Anything we can address separately, @rcoll or @Automattic/jetpack-martech?